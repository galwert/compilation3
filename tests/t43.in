
int func() {
    return 1;
}

int ags() {
    return 1;
}

int zxvc() {
    return 1;
}

int sdg() {
    return 1;
}

int qwr(bool x1, bool x2) {
    return 1;
}
void main(){
    printi(sdg());
}


bool sdfa4(bool x1, bool x2) {
    return x1 or x2;
}

void nbfdrct(bool x1, bool x2) {
    int x = 1;
    byte y = 2 b;

    return;
}

int cwrvsgdhs(bool x1, bool x2) {
    return 1;
}

int ytyjbd(bool x1, bool x2) {
    return 1;
}

int afsdtv(bool x1, bool x2) {
    int x3;
    byte x4;
    x4 = 10; // should fail here - assing int to byte
    return 1;
}

