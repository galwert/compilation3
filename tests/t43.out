---end scope---
---end scope---
---end scope---
---end scope---
---end scope---
x1 BOOL -1
x2 BOOL -2
---end scope---
---end scope---
x1 BOOL -1
x2 BOOL -2
---end scope---
x1 BOOL -1
x2 BOOL -2
x INT 0
y BYTE 1
---end scope---
x1 BOOL -1
x2 BOOL -2
---end scope---
x1 BOOL -1
x2 BOOL -2
line 48: type mismatch
